src/core/connection.c

```diff
@@ -6602,6 +6602,30 @@ QuicConnParamSet(
         return QUIC_STATUS_SUCCESS;
     }
 
+    case QUIC_PARAM_CONN_THIS_RELIABLE_RESET_ENABLED:
+
+        if (BufferLength != sizeof(BOOLEAN)) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+
+        if (QUIC_CONN_BAD_START_STATE(Connection)) {
+            Status = QUIC_STATUS_INVALID_STATE;
+            break;
+        }
+
+        Connection->Settings.ThisReliableResetFrameEnabled = *(BOOLEAN*)Buffer;
```

Comment:
No need to include "Frame" in any of these variable names. Name it after the feature, which is support for ReliableStreamReset.